pkgs/data/fonts/intel-one-mono/default.nix

```
@@ -0,0 +1,33 @@
+{ lib, stdenvNoCC, fetchzip }:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "intel-one-mono";
+  version = "1.2.0";
+
+  src = fetchzip {
+    url = "https://github.com/intel/intel-one-mono/releases/download/V${version}/ttf.zip";
+    sha256 = "sha256-uZRRiwjBr3Cm10uh0ji65+ipSkKSKa8VOx8ZVrZOJKI=";
+    stripRoot = false;
+  };
+
+  dontPatch = true;
+  dontConfigure = true;
+  dontBuild = true;
+  doCheck = false;
+  dontFixup = true;
+
+  installPhase = ''
+    runHook preInstall
+    install -Dm644 -t $out/share/fonts/truetype/ ttf/*.ttf
+    runHook postInstall
+  '';
+
+  meta = with lib; {
+    description = "Intel One Mono, an expressive monospaced font family that’s built with clarity, legibility, and the needs of developers in mind.";
```

You should remove the dot at the end.
```suggestion
    description = "Intel One Mono, an expressive monospaced font family that’s built with clarity, legibility, and the needs of developers in mind";
```